src/components/Browser/RequestsProcessor/GetBalance/index.js

```diff
@@ -15,10 +15,18 @@ export default function makeGetBalance() {
   return compose(
```

Comment:
You'll need to update `makeGetBalances` to accept `balancesActions` instead, and then within `mappings.js` include `makeBalancesActions` there.